src/main/java/com/epam/lab/arrayscollections1/stringcontainer4/Main.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.lab.arrayscollections1.stringcontainer4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Main {
+    private static Logger logger1 = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args){
+
+        long startTime = System.nanoTime();
+        MyStringContainer myStringContainer = new MyStringContainer();
+
+        myStringContainer.add("A");
```

Comment:
I would implement additional method add that accepts var args or list add(String ... values){ Stream.of(values).forEach(v -> this::add) }

as a result, will shorten code for few lines